Explorer++/Explorer++/Storage.cpp

```diff
@@ -5,13 +5,38 @@
 #include "stdafx.h"
 #include "Storage.h"
 #include "../Helper/ProcessHelper.h"
+#include <wil/win32_helpers.h>
 #include <filesystem>
 
 namespace Storage
 {
 
+std::optional<std::wstring> GetExpandedEnvironmentVariable(const std::wstring &name)
+{
+	wil::unique_hglobal_string value;
+	HRESULT hr = wil::GetEnvironmentVariableW(name.c_str(), value);
+	if (FAILED(hr))
+	{
+		return std::nullopt;
+	}
+
+	wil::unique_hglobal_string expandedValue;
+	hr = wil::ExpandEnvironmentStringsW(value.get(), expandedValue);
+	if (FAILED(hr))
+	{
+		return std::nullopt;
+	}
+	return std::wstring(expandedValue.get());
+}
+
 std::wstring GetConfigFilePath()

```

Comment:
I think it would also be useful to add at least one unit test for this that checks that `GetConfigFilePath` returns the value in `EXPLORERPP_CONFIG`, if that value is set. The test should be very simple - just assign some value to `EXPLORERPP_CONFIG` and verify that `GetConfigFilePath` returns that value.